include/argparse/argparse.hpp

```diff
@@ -683,7 +683,7 @@ class Argument {
     auto hspace = "  "; // minimal space between name and help message
     stream << name_stream.str();
     std::string_view help_view(argument.m_help);
-    while ((pos = argument.m_help.find('\n', prev)) != std::string::npos) {
+    while ((pos = argument.m_help.find('\n', prev)) != (signed)std::string::npos) {
```

Comment:
Instead of c-style casting npos to signed I would change `auto pos = 0` to `size_t pos = 0`.  The root problem is that "pos" is being auto-typed to an int whereas the return type of string::find() and the type of string::npos are both size_t.

Reply:
It also gets auto-typed correctly if you mark the zeros as unsigned:
``` C++
    auto pos = 0u;
    auto prev = 0u;
```

Reply:
I resolved/suppressed the warning by using a static_cast<int>, my minimal reasoning was that the std::string::npos will always be a signed int (of -1) according to the std::string::npos https://cplusplus.com/reference/string/string/npos/
``while ((pos = argument.m_help.find('\n', prev)) != static_cast<int>(std::string::npos)) {``